src/github.com/matrix-org/go-neb/services/rssbot/rssbot.go

```diff
@@ -348,12 +348,17 @@ func (s *Service) sendToRooms(cli *gomatrix.Client, feedURL string, feed *gofeed
 	return nil
 }
 
-// SomeOne posted a new article: Title Of The Entry ( https://someurl.com/blag )
 func itemToHTML(feed *gofeed.Feed, item gofeed.Item) gomatrix.HTMLMessage {
-	return gomatrix.GetHTMLMessage("m.notice", fmt.Sprintf(
-		"<i>%s</i> posted a new article: %s ( %s )",
-		html.EscapeString(feed.Title), html.EscapeString(item.Title), html.EscapeString(item.Link),
-	))
+	return &gomatrix.HTMLMessage{
+		Body: fmt.Sprintf("%s: %s (%s)",
+			html.EscapeString(feed.Title), html.EscapeString(item.Title), html.EscapeString(item.Link)),
+		MsgType: "m.notice",
+		Format: "org.matrix.custom.html",
+		FormattedBody: fmt.Sprintf("<strong>%s</strong>:<br><a href=\"%s\"><strong>%s</strong></a>",
```

Comment:
An assumption I'm making here is that no URLs will have `"` in them, because I'm just interpolating.

It would be best to run the URL through a URL encoder that will escape the `"`, or even just substitute `"` with the proper URL escape code with crude string substitution, but I don't know what libraries I might use to do those things in Go, or exactly how to do it since I've never written a line of Go before.

Reply:
Does `html.EscapeString()` work for this purpose?

Reply:
Looking at https://golang.org/pkg/html/#EscapeString , it looks like this will insert HTML escape codes into our URL, but we want URI escape codes.

Is there a Go REPL I can use to test this without having to set up and run my own go-neb?

Reply:
I did some reading: HTML escape codes in HTML attribute values are indeed valid and are the proper way to escape text inside attribute values!

Testing the following code in https://play.golang.org, it looks to produce output valid by that:
```go
package main

import (
	"fmt"
	"html"
)

func main() {
	fmt.Printf("<a href=\"%s\">World</a>", html.EscapeString("https://example.com/<\"\">?this=that&that=this"))
}
```

Reply:
Looks good.

P.S. Heads up you can share a play session so people can run the code already loaded in: https://play.golang.org/p/TgSLH8n-JyT